libs/sdk-ui-charts/src/highcharts/chart/highcharts/customConfiguration.ts

```diff
@@ -1112,6 +1112,7 @@ function getAxesConfiguration(chartOptions: IChartOptions) {
             );
 
             const tickConfiguration = getXAxisTickConfiguration(chartOptions);
+            const minRange = get(chartConfig, "chart.zoomType", undefined) ? 2 : undefined;
```

Comment:
Please add contant for `2` and comment its purpose

Reply:
Done!